app/views/handlers/feedback.py

```diff
@@ -109,13 +109,14 @@ def question_schema(self):
                         {
                             "label": lazy_gettext("The census questions"),
                             "value": lazy_gettext("The census questions"),
+                            "description": lazy_gettext(
+                                "For example, question not clear, answer option not relevant"
```

Comment:
I'm not sure which text is correct on the card - the description has "For example, question not clear, answer option not relevant", but the image shows "For example, questions not clear, answer options not relevant".

Reply:
I spoke to @jrbarnes9 and question/answer should be plural as you pointed out. Card updated and PR fixed!